modules/nback.llvm/tests/utils.bal

```diff
@@ -0,0 +1,21 @@
+import ballerina/io;
+
+function buildOutput(Module module, string outputPath) returns error? {
+    Output out = check new (outputPath);
+    module.output(out);
+    check out.finish();
+}
+
+function compareFiles(string expectedFilePath, string resultFilePath) returns boolean|error {
```

Comment:
Using io:fileReadLines instead of the byte channel stuff

Reply:
Fixed